src/app/tabs/run-tab.js

```diff
@@ -96,39 +98,66 @@ function runTab (opts, localRegistry) {
   var container = yo`<div class="${css.runTabView}" id="runTabView" ></div>`
   var recorderInterface = makeRecorder(localRegistry, event, self)
 
-  self._view.collapsedView = yo`
-    <div class=${css.recorderCollapsedView}>
-      <div class=${css.recorderCount}>${self._view.recorderCount}</div>
-    </div>`
+  self._view.recorderCard = {
+      collapsedView: yo`
+        <div class=${css.recorderCollapsedView}>
+          <div class=${css.recorderCount}>${self._view.recorderCount}</div>
+        </div>`,
+      expandedView: yo`
+        <div class=${css.recorderExpandedView}>
+          <div class=${css.recorderDescription}>
+            ${self.data.text}
+          </div>
+          <div class="${css.transactionActions}">
+            ${recorderInterface.recordButton}
+            ${recorderInterface.runButton}
+            </div>
+          </div>
+        </div>`
+  }
+  self._view.rpcResponsePre = opts.useSpan ? yo`<span id="rpcResponsePre" />` : yo`<pre id="rpcResponsePre" />`
+  self._view.rpcResponseCard = {
+    collapsedView: yo`
+     <div class=${css.recorderCollapsedView} id="rpcResponseContainer">
+     </div>`,
+    expandedView: yo`
+     <div class=${css.recorderExpandedView} id="rpcResponseContainer">
+       <div class=${css.recorderDescription} id="rpcResponsePreContainer">
+         ${self._view.rpcResponsePre}
+       </div>
+       <div class="${css.transactionActions}">
+         ${copyToClipboard(() => self._view.rpcResponsePre.value)}
+       </div>
+     </div>`
+  }
+  function expandCollapseCard (cardView) {
+    return function (arrow, body, status) {
+      body.innerHTML = ''
+      status.innerHTML = ''
+      if (arrow === 'down') {
+        status.appendChild(cardView.collapsedView)
+        body.appendChild(cardView.expandedView)
+      } else if (arrow === 'up') {
+        status.appendChild(cardView.collapsedView)
+      }
+    }
+  }
 
-  self._view.expandedView = yo`
-    <div class=${css.recorderExpandedView}>
-      <div class=${css.recorderDescription}>
-        ${self.data.text}
-      </div>
-      <div class="${css.transactionActions}">
-        ${recorderInterface.recordButton}
-        ${recorderInterface.runButton}
-        </div>
-      </div>
-    </div>`
+  self.rpcResponseOpts = {
```

Comment:
I think that the rpc response should not be in the main panel, that take too much space. "signing message" is still a side feature of this tab. But thanks for giving a try that way.
would it be possible to display the result 1) in an another modal 2) I think only the hex string value need to be displayed.